packages/devtools_app/test/test_infra/scenes/standalone_ui/mock_editor_widget.dart

```diff
@@ -54,12 +71,19 @@ class _MockEditorWidgetState extends State<MockEditorWidget> {
   void initState() {
     super.initState();
 
-    logUpdated = editor.log.map((log) {
-      logRing.add(log);
-      while (logRing.length > maxLogEvents) {
-        logRing.removeFirst();
+    // Listen to the log streams to maintain our buffer and trigger rebuilds.
+    clientLogUpdated = clientLog.map((log) {
+      clientLogRing.add(log);
+      while (clientLogRing.length > maxLogEvents) {
+        clientLogRing.removeFirst();
+      }
+    }).asBroadcastStream();
+    editorLogUpdated = editorLog.map((log) {
+      editorLogRing.add(log);
+      while (editorLogRing.length > maxLogEvents) {
+        editorLogRing.removeFirst();
       }
-    });
+    }).asBroadcastStream();
```

Comment:
do these stream subs need to be cancelled? if so you can use the AutoDisposeMixin to add then as automatically disposed subscriptions

Reply:
Done in d50c807fe